### ISSUE TITLE
If verbose_eval is int, any verbose_eval would be '1'

### ISSUE THREAD

**Issue by KazukiOnodera** [dmlc/xgboost#6516]
In 1.3.0, if verbose_eval is int, any verbose_eval would be '1'.
e.g. If verbose_eval=100 was passed, the evaluation metric on the validation set should be printed at every '100' boosting stage, but actually the evaluation metric would be printed at every '1' boosting stage.
I confirmed this doesn't happen in 1.2.0.

**Comment by trivialfis** [comment]
Yup, it's fixed in master branch.  For now,  you can consider getting a nightly build by the link listed in installation doc.
